### ISSUE TITLE
Question: labelling different activities

### ISSUE THREAD

**Issue by yschimke** [usefulness/easylauncher-gradle-plugin#298]
Can you label the icon with the activity name? For a library with a few sample activities for testing different things, it would be great to be able to label each one distinctly without generating a new icon for each.

**Comment by mateuszkwiecinski** [comment]
Hey 👋 
I'm not sure if that's feasible 🤔 Or at least, I don't see a way to achieve such effect with current plugin capabilities. It _feels_ like the plugin works on a "different level", it plugs into the build process to generate icons for different build variants. In the case of multiple activities, they all will use a single build variant. The fact of using different build variants is crucial, as the core principal how this plugin works is it provides "more specific" resource than the _default_ one. (given there is an icon defined as `main/res/xxx.webp` resource, the plugin generates new resource under `debug/res/xxx.webp` which then is picked by AGP).  
In your case - it's not possible to rely on the described behaviour as all activities use the same build variant. To make your use case work - in generated code you'd have to either override `AndroidManifest` entries (I don't know if that's possible) or modify source files to point at a generated icon.

tl;dr I don't even know if that's feasible - it'd have to rely on a completely different mechanism than what this plugin currently does, so it either can't be done or can't be easily integrated with the plugin.

Just out of curiosity: how many activities do you have?

**Comment by yschimke** [comment]
Only 3 - so easy to fix one off.  But more that as I'm not an icon designer, just being able to label them is way more useful for the test activities.

Thanks, thought it worth asking.
